pom.xml

```diff
@@ -62,6 +62,11 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-processor</artifactId>
+            <version>${mapstruct.version}</version>
+        </dependency>
```

Comment:
Why do you duplicate this dependency?

Reply:
It's not duplicate. I tried to handle with mappers, and there was a solution to add mapstruct-processor dependency in some forum.